static/js/index.js

```diff
@@ -0,0 +1,147 @@
+const API_PREFIX = 'api';
+const baseUrl = window.location.origin;
+
+const setScrapingLogs = (logs) => {
+    const logsNode = document.getElementById('scraping-logs')
+    for (i in logs) {
+        pNode = document.createElement('p')
+        pNode.innerText = logs[i]
```

Comment:
You can use a string interpolation to create the p element with the text in one go.